homeassistant/components/anova/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Config flow for Anova."""
+from __future__ import annotations
+
+from anova_wifi import AnovaApi, AnovaOffline, InvalidLogin, NoDevicesFound
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN
+
+
+class AnovaConfligFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Sets up a config flow for Anova."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, str] | None = None
+    ) -> FlowResult:
+        """Handle a flow initiated by the user."""
+        errors: dict[str, str] = {}
+        if user_input is not None:
+            api = AnovaApi(
+                aiohttp_client.async_get_clientsession(self.hass),
+                user_input[CONF_USERNAME],
```

Comment:
Unless there is an account id, you can use the normalized unique id as the unique id

https://developers.home-assistant.io/docs/config_entries_config_flow_handler?_highlight=unique&_highlight=id#sometimes-acceptable-sources-for-a-unique-id-for-cloud-services

Reply:
In the api calls I have come across - I have not seen it, so for now, I am making it the username lowercase